sr_description/hand/xacro/forearm/forearm.urdf.xacro

```diff
@@ -91,6 +89,20 @@
           </geometry>
         </xacro:unless>
       </collision>
+      <xacro:if value="${cable_mesh}">
+        <visual>
+          <origin rpy="0 -1.57 -1.57" xyz="0.0 0.067 0.0045"/>
+          <geometry name="${prefix}cable_extr_collision">
```

Comment:
I think the name of the geometries is inverted: this one should be "visualization"